triveni-codev/GBWebApp-dev
# Darlp service map networks 1205458303586175

@triveni-codev , this is already implemented and all set of concerns  (physicalID, BSID, transport array reference) were already addressed. This is already implemented in both ATSC3 and non ATSC3 (Cable or Terrestrial).

ASANA ticket: https://app.asana.com/0/1201943683850147/1205458303586175

To check: 
-create an atsc3 type of transport with ip streams then link that ip stream to a service in network (you should be able to see it under physical transports of service map)
-create a non-atsc3 type of transport (cable or terrestrial) with a program then link that ip stream to a service in network (you should be able to see it under physical transports of service map)

Notes: 
-atsc3 and non-atsc3 have different display in service map under physical transports
-list of transport is from the client transports model since they are always from server
-for concerns on clicking a service but didn't render the right service map, please don't log it as it's not part of this ticket's scope


Additional testing:
ASANA ticket: https://app.asana.com/0/1201943683850147/1205458303586191

To test: 
-in dtv-networks, make sure that transports tab is active and the display is already auto populated (before it has no display)


Additional testing:
ASANA ticket: https://app.asana.com/0/1201943683850147/1204184790043264

To test:
-make sure that "dtv-service" files/folders/verbiage before is already changed to "dtv-services" w/o any further errors while using the app.
